src/controllers/Room.controller.js

```diff
@@ -0,0 +1,7 @@
+const RoomService = require('../services/Room.service.js');
+
+const getAll = (req, res) => {
+  res.status(200).send(RoomService.rooms);
```

Comment:
This is not good practice to send the service functions directly. Use `trycatch` to handle possible errors.